### ISSUE TITLE
HTML5 Video is becoming a big issue.

### ISSUE THREAD

**Issue by zacxander** [qutebrowser/qutebrowser#4100]
I can't watch a lot of youtube videos and most videos on other sites aren't working, I've tried installing the DLLs with no result.

**Comment by The-Compiler** [comment]
I assume this is on Windows? You'll get proprietary codec support if you recompile Qt from sources, which is not something I'm willing to do for the release binaries (as compiling big stuff on Windows is a pain).

**Comment by jasonstone20** [comment]
I have the same issue running GalliumOS Linux on a Lenovo N21 Chromebook.  I have installed gstreamer0.10-good, x264, gstreamer0.10-ffmpeg, and a few others.  I can get almost all YouTube videos to run by use mpv as my video player.  This is an awesome browser, it is just what I have been looking for and wanting for years.

**Comment by The-Compiler** [comment]
@jasonstone20 GStreamer is only relevant for QtWebKit, not the default QtWebEngine backend. If you installed qutebrowser/QtWebEngine via GalliumOS packages, they probably build without proprietary codec support as well. If you use a tox install, it uses the official PyQt/Qt release binaries which come without proprietary codecs as well.

**Comment by jasonstone20** [comment]
Yes, I used the tox install.  That makes sense why nothing I tried worked.  Do you know if there is somewhere I can get the codecs?  Thank you for your response also.

**Comment by The-Compiler** [comment]
@jasonstone20 You can't get codecs separately with QtWebEngine - either your QtWebEngine is built with them enabled, or it isn't. You'd need to use the GalliumOS QtWebEngine (if built with those enabled), or build it from sources yourself.

**Comment by jasonstone20** [comment]
Thank you, I just installed it and will give it a shot.

**Comment by jasonstone20** [comment]
I gave it a try by first just installing the QtWebEngine on top of my already install qutebrowser, and that didn't seem to work.  So I deleted the qutebrowser dircetory, installed the QtWebEngine, then used the tox install.  Looks like it worked, I can play most videos now.  On youtube/html5 the only boxes not checked are the h264 and h264 & mse.  Befor only two boxes were check out of six, now its up to four.
